File: 8-kyu/java/calculate-bmi/calculate-bmi.java
public class Calculate {
  public static String bmi(double weight, double height) {
    Double ind = weight / (height * height);
    if (ind <= 18.5) {return "Underweight";}
    else if (ind <= 25.0) {return "Normal";}
    else if (ind <= 30.0) {return "Overweight";}
    else {return "Obese";}
  }
}

File: 8-kyu/python/calculate-bmi/calculate-bmi.py
def bmi(weight, height):
    
    ind = weight / (height ** 2)
    match ind:
        case _ if ind <= 18.5 : return "Underweight"
        case _ if ind <= 25.0 : return "Normal"
        case _ if ind <= 30.0 : return "Overweight"
        case _ if ind > 30 : return "Obese"
    